.github/workflows/backend-test.yml

```
@@ -0,0 +1,68 @@
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: "npm"
+
+      - run: npm ci --workspaces
+
+      - name: Wait for Postgres
+        run: |
+          sudo apt-get install -y postgresql-client
+          until pg_isready -h localhost -p 5432 -U postgres; do 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)